app/services/mark_patient_mobile_numbers.rb

```diff
@@ -0,0 +1,26 @@
+class MarkPatientMobileNumbers
+  def self.call
+    new.call
+  end
+
+  def call
+    return unless ENV["FORCE_MARK_PATIENT_MOBILE_NUMBERS"] == "true"
+
+    count = eligible_numbers.count
+    notify("Force-marking #{count} as mobile numbers")
+
+    eligible_numbers.update_all(phone_type: "mobile")
```

Comment:
There's 12k phone numbers that this will update in a single query everyday. Should we consider batching, or do you think this suffices for now?

Reply:
It's ~7k in BD today. It's a single SQL query that won't stress app memory or performance. I think a single sweep is ok.